doc/langref.html.in

```diff
@@ -7831,35 +7831,19 @@ comptime {
       {#header_close#}
 
       {#header_open|@atomicRmw#}
-      <pre>{#syntax#}@atomicRmw(comptime T: type, ptr: *T, comptime op: builtin.AtomicRmwOp, operand: T, comptime ordering: builtin.AtomicOrder) T{#endsyntax#}</pre>
+      <pre>{#syntax#}@atomicRmw(comptime T: type, ptr: *T, comptime op: std.builtin.AtomicRmwOp, operand: T, comptime ordering: std.builtin.AtomicOrder) T{#endsyntax#}</pre>
       <p>
-      This builtin function atomically modifies memory and then returns the previous value.
+      This builtin function atomically modifies memory and then returns the previous value - "Read-Modify-Write".
```

Comment:
What about: "This builtin function, short for "Read Modify Write", atomically ..."

Reply:
I would rather get rid of "This builtin function" as that is completely redundant and this is one of the few places where that's done. This style of explaining an acronym or abbreviation is used for `@popCount` too:

> Counts the number of bits set in an integer - "population count".

I think in this PR I'll stick with it for now.